src/dynamo.py

```diff
@@ -61,6 +62,44 @@ def _scan_with_condition(
     return items
 
 
+def is_receiver_available(receiver: str):
```

Comment:
I feel like this name doesn't fully express what this wants to do. Maybe
`has_ receiver_opted_out` ?

Reply:
changed it :smile: